docs/api/json.md

```diff
@@ -0,0 +1,291 @@
+# JSON Format
+
+Feldera can ingest and output data in the JSON format
+- via [`/ingress`](/api/push-data-to-a-sql-table) and
+  [`/egress`](/api/subscribe-to-a-stream-of-updates-from-a-sql-view-or-table)
+  REST endpoints, or
+- as a payload received from or sent to a connector
+
+Here we document the JSON format supported by Feldera.  The specification
+consists of four parts:
+
+1. [Encoding invividual table rows](#encoding-individual-rows). Describes
```

Comment:
I'd give some example here for all 4 so a dev can immediately see what each one is about and jump to the right section. From my experiments, they don't typically read everything, jsut browse and jump

Reply:
great idea!